sdk/test-utils/perf/src/eventPerfTest.ts

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AbortController, AbortSignalLike } from "@azure/abort-controller";
+import { PerfTestBase } from "./perfTestBase";
+import { isDefined } from "./utils";
+
+/**
+ * Extends PerfTestBase, enables writing perf tests for the APIs that receive events as a stream
+ * - Typically, such APIs("subscribe" method) offered by the AMQP based SDKs - event-hubs/service-bus.
+ */
+export abstract class EventPerfTest<
+  TOptions = Record<string, unknown>
+> extends PerfTestBase<TOptions> {
+  start = process.hrtime();
```

Comment:
Does `start` need to be set in the field initializer or the ctor, if it's also set at the start of `runAll()`?

Reply:
removed